site/assets/js/snippets.js

```diff
@@ -75,28 +75,35 @@
   // -------------------------------
   // Alerts
   // -------------------------------
-  // Used in 'Show live alert' example in docs or StackBlitz
+  // Used in 'Show live toast' example in docs or StackBlitz
 
-  // Boosted mod: adapted innerHTML to have the icon and so added a parameter within `alert()` function
+  // Boosted mod: adapted innerHTML to have the icon and so added a parameter within alert() and a tooltip on btn-close
   // js-docs-start live-alert
   const alertPlaceholder = document.getElementById('liveAlertPlaceholder')
-  const alert = (message, type, typeVisuallyHidden) => {
+  function alert(message, type, typeVisuallyHidden) {
```

Comment:
In order to be the closest as possible of Bootstrap

```suggestion
  const appendAlert = (message, type, typeVisuallyHidden) => {
```

And then use `appendAlert` when it's called right after like:

https://github.com/twbs/bootstrap/blob/7d97fdff46718ecbb3fa33d77384d827870c9bc9/site/assets/js/snippets.js#LL97C7-L97C18

And in the "Boosted mod" comment above